stdlib/3.4/statistics.pyi

```diff
@@ -0,0 +1,23 @@
+# Stubs for statistics
+
+from decimal import Decimal
+from fractions import Fraction
+from typing import Iterable, TypeVar
+
+# Most functions in this module accept homogeneous collections of one of these types
+_Number = TypeVar(int, float, Decimal, Fraction)
+
+class StatisticsError(ValueError): ...
+
+def mean(data: Iterable[_Number]) -> _Number: ...
```

Comment:
In fact, median for ints returns a float if the number of elements is even. So I think we're good :)